src/test/regress/expected/isolation_ensure_dependency_activate_node.out

```diff
@@ -1448,6 +1727,383 @@ run_command_on_workers
+	SELECT create_distributed_function('add(INT,INT)');
+ <waiting ...>
+step s1-commit: 
+    COMMIT;
+
+step s2-distribute-function: <... completed>
+create_distributed_function
+
+               
+step s2-print-distributed-objects: 
+    -- print an overview of all distributed objects
+    SELECT pg_identify_object_as_address(classid, objid, objsubid) FROM citus.pg_dist_object ORDER BY 1;
+
+    -- print if the schema has been created
+    SELECT count(*) FROM pg_namespace where nspname = 'myschema';
+    SELECT run_command_on_workers($$SELECT count(*) FROM pg_namespace where nspname = 'myschema';$$);
+
+    -- print if the type has been created
+	SELECT count(*) FROM pg_type where typname = 'tt1';
+    SELECT run_command_on_workers($$SELECT count(*) FROM pg_type where typname = 'tt1';$$);
+
+    -- print if the function has been created
+    SELECT count(*) FROM pg_proc WHERE proname='add';
+    SELECT run_command_on_workers($$SELECT count(*) FROM pg_proc WHERE proname='add';$$);
+
+pg_identify_object_as_address
+
+(function,"{public,add}","{integer,integer}")
+count          
+
+0              
+run_command_on_workers
+
+(localhost,57637,t,0)
+(localhost,57638,t,0)
+count          
+
+0              
+run_command_on_workers
+
+(localhost,57637,t,0)
+(localhost,57638,t,0)
+count          
+
+1              
+run_command_on_workers
+
+(localhost,57637,t,1)
+(localhost,57638,t,1)
+master_remove_node
+
+               
+               
+
+starting permutation: s1-print-distributed-objects s1-begin s2-public-schema s2-distribute-function s1-add-worker s1-commit s2-print-distributed-objects
+?column?       
+
+1              
+step s1-print-distributed-objects: 
+    SELECT 1 FROM master_add_node('localhost', 57638);
+
+    -- print an overview of all distributed objects
+    SELECT pg_identify_object_as_address(classid, objid, objsubid) FROM citus.pg_dist_object ORDER BY 1;
+
+    -- print if the schema has been created
+    SELECT count(*) FROM pg_namespace where nspname = 'myschema';
+    SELECT run_command_on_workers($$SELECT count(*) FROM pg_namespace where nspname = 'myschema';$$);
+
+    -- print if the type has been created
+	SELECT count(*) FROM pg_type where typname = 'tt1';
+    SELECT run_command_on_workers($$SELECT count(*) FROM pg_type where typname = 'tt1';$$);
+
+    -- print if the function has been created
+    SELECT count(*) FROM pg_proc WHERE proname='add';
+    SELECT run_command_on_workers($$SELECT count(*) FROM pg_proc WHERE proname='add';$$);
+
+    SELECT master_remove_node('localhost', 57638);
+
+?column?       
+
+1              
+pg_identify_object_as_address
+
+count          
+
+0              
+run_command_on_workers
+
+(localhost,57637,t,0)
+(localhost,57638,t,0)
+count          
+
+0              
+run_command_on_workers
+
+(localhost,57637,t,0)
+(localhost,57638,t,0)
+count          
+
+0              
+run_command_on_workers
+
```

Comment:
I think we should not see `1` here  as well. But, since dropping the function doesn't remove item from `pg_dist_object`, we end-up with `1` because `master_add_node`  which is called from `print-distributed-objects` -- that's weird btw -- replicates the function.

Maybe for now, just remove it  manually from `pg_dist_object`, and once your DROP FUNCTION pr would do it automatically, we'd remove the manual removal.



So, have the following next to `DROP FUNCTION` call
```SQL

DELETE FROM citus.pg_dist_object WHERE objid = 'add(int,int)'::regprocedure;
```